pkg/monitor/monitor/util.go

```diff
@@ -341,26 +345,6 @@ func getMonitorInitContainer(monitor *v1alpha1.TidbMonitor, tc *v1alpha1.TidbClu
 		},
 		Resources: controller.ContainerResource(monitor.Spec.Initializer.ResourceRequirements),
 	}
-	if tc != nil {
-		container.Env = append(container.Env, []core.EnvVar{
-			{
-				Name:  "TZ",
-				Value: tc.Timezone(),
-			},
-			{
-				Name:  "TIDB_ENABLE_BINLOG",
-				Value: strconv.FormatBool(tc.IsTiDBBinlogEnabled()),
-			},
-			{
-				Name:  "TIDB_CLUSTER_NAME",
-				Value: tc.Name,
-			},
-			{
-				Name:  "TIDB_CLUSTER_NAMESPACE",
```

Comment:
This cannot be removed now, and is being used in https://github.com/pingcap/monitoring/blob/master/platform-monitoring/operator/init.sh#L116, we need to update the pods.json to support multiple clusters before removing this.